lib/project_types/script/layers/infrastructure/assemblyscript_project_creator.rb

```diff
@@ -10,6 +10,7 @@ class AssemblyScriptProjectCreator
         property! :script_name, accepts: String
         property! :path_to_project, accepts: String
 
+        MIN_NODE_VERSION = "14.5.0"
```

Comment:
I'd prefer if we could keep this in an AS specific project file but we don't have that right now so I thought to keep it here. I have some refactor plans that would fix that.

Also side note - anyone know why this node version is so high? I think the latest version is 14.5.1. Do we really need to rely on such a recent version? 🤔

Reply:
Node 14.5.1 is the latest long-term support version and is bundled with NPM. I think we should be supporting recent LTS versions as a minimum to ensure the user has up-to-date bug/security fixes with NPM.